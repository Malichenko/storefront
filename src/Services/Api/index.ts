import * as ApiServices from './index-all';
export { ApiServices };
