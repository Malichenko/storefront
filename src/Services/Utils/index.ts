import * as UtilServices from './index-all';

export { UtilServices };
