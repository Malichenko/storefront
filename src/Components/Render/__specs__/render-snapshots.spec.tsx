import React from 'react';
import { TestRenderComponent, TestsService } from 'Services/Tests';
import { Render, RenderProps } from '../index';

const renderComponent = (props: RenderProps): TestRenderComponent =>
	TestsService.renderComponent(
		<Render {...props}>
			<div>Lorem ipsum dolor sit amet.</div>
		</Render>
	);

describe('<Render />', () => {
	test('Should render children', () => {
		const component = renderComponent({ if: true });
		expect(component).toMatchSnapshot();
	});
	test('Should NOT render children', () => {
		const component = renderComponent({ if: false });
		expect(component).toMatchSnapshot();
	});
});
