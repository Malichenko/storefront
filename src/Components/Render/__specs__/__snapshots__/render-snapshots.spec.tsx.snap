// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Render /> Should NOT render children 1`] = `
<div>
  HeaderComponent
</div>
`;

exports[`<Render /> Should render children 1`] = `
<div>
  HeaderComponent
</div>
`;
